### ISSUE TITLE
Mass deletion of mod downloads

### ISSUE THREAD

**Issue by Sexyowl** [ModOrganizer2/modorganizer#805]
[0a75ca2be7692cb881a0076a0d16466a-b7e667e5b249bcb8923f76cc16fdb2367e160b27.zip](https://github.com/ModOrganizer2/modorganizer/files/3459445/0a75ca2be7692cb881a0076a0d16466a-b7e667e5b249bcb8923f76cc16fdb2367e160b27.zip)
[0a75ca2be7692cb881a0076a0d16466a-b7e667e5b249bcb8923f76cc16fdb2367e160b27.zip](https://github.com/ModOrganizer2/modorganizer/files/3459447/0a75ca2be7692cb881a0076a0d16466a-b7e667e5b249bcb8923f76cc16fdb2367e160b27.zip)

## The problem:
Similar to Issue #657, MO2 deleted some of my downloads (about 11 gigs) and started showing only 12. Of these 3 were installed and the rest had not been. I have over 167 mods installed and more which were not activated. However, it did not uninstall them. Since then, it has not been reading the downloads folder even though the path is correct.

## To Reproduce: 

1. Go to : Open MO2 and click the downloads tab on the right pane?

## Environment:

* Mod Organizer Version that exhibits the issue: 2.2.1
* Last Mod Organizer Verison that did not exhibit the issue (if applicable): 2.2.1
* Desktop OS/version used to run Mod Organizer: Windows 7

## Details:
If necessary, describe the problem you have been experiencing with more details and pictures.

## Link to Mod Organizer logs:
To access the log files you can press the Open folder icon over the modlist and select "Open MO2 Logs folder". You should close MO2 before sending or examining the logs as they are finished writing only after the program is closed.

### USVFS:
Create a [GIST](https://gist.github.com) which is a paste of your USVFS logs, and link them here.
Do _NOT_ paste your full USVFS logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the USVFS logs!

### MO Interface:
Create a [GIST](https://gist.github.com) which is a paste of your mo_interface logs, and link them here.
Do _NOT_ paste your full mo_interface logs here, as it will make this issue very long and hard to read!
If you are reporting a bug, _always_ include the mo_interface logs!

**Comment by isanae** [comment]
Are the downloads actually deleted from your hard drive or are they just not shown in MO?

**Comment by Sexyowl** [comment]
Some were deleted (i.e. sent to the recycle bin) while some weren't. The
ones which were left are not showing up in MO, but I can see them in the
folder. The plugins themselves were not deleted.

**Comment by SheiFoxy** [comment]
My sister and I are both having this issue.
For us, it's deleting the oldest mods we downloaded so we're slowly losing our mod downloads into the recycle bin.
I've never had this happen before the latest version.

**Comment by Silarn** [comment]
Something none of you have provided yet are logs.

**Comment by Sexyowl** [comment]
Yes I did. You asked for Gist and that’s what I uploaded. Recheck before you accuse please.

**Comment by LostDragonist** [comment]
We might both be blind but where?  The place where they'd normally go is unmodified from the issue template: 
![image](https://user-images.githubusercontent.com/7862539/62659174-4218c880-b930-11e9-9d8c-22e0e2a48d1a.png)

**Comment by Silarn** [comment]
It's in the zip links at the top of the ticket.

**Comment by Silarn** [comment]
The attached mo_inferface.txt doesn't appear to include any downloads, just the base initialization of MO2.

You'd need to replicate the issue, close MO2, and then upload the current interface file.

**Comment by Sexyowl** [comment]
Fist of all, I went to the GIST page on github, uploaded the logs and asked it to create the GIST. Those zip files are what it gave me. When I tried to paste them where you are indicating it placed it where you see it.

@Silarn - how do I "replicate" the issue? I can't tell it to re-delete files it has already deleted. At present its showing 12 files as you can see in the .png below, which means its not reading the rest from the downloads folder. 
I read the instructions before posting and that was the log I got when I closed MO2.

![Downloads](https://user-images.githubusercontent.com/20115300/62665185-6c787f00-b94c-11e9-8112-eab22d32e3d3.png)

**Comment by Silarn** [comment]
I'm actually surprised it let you upload a zip, gist is intended to take a text file of some sort to view it in the browser.

Anyway, what you're telling us is very unclear. If your downloads are deleted, then we need a log where that is happening, when it happens. If they're just not showing up, then a) make sure the files are actually in the download directory and have proper read/write permissions. If so, then b) try showing hidden downloads in the download pane just to see what happens.

**Comment by Sexyowl** [comment]
Sorry for the late reply...was away from my PC for a few days. Since the initial deletion MO2 has not done so again, so I cannot post the log you want. The files which were not deleted are still in the downloads folder and am logged in as administrator with full permissions. Checking the hidden downloads shows only those files which I hid (actually all from 2018).  And yes the meta files are there. 

I'll give you example of what I mean...if you look at the screenshot I posted, you'll see a mod Gotobed (v1.5.1) installed on July 29th. This mod was there previously and I was using it before this issue. However, when I went to install an update (on July 29th) I could not find it in the downloads pane nor was it in the downloads folder. The plugin was there because I was using it in the game. It only showed up in the folder and downloads pane when I downloaded and installed the update.

Actually this was when I realised what had occurred. I use CCleaner and remember my Recycle Bin having over 11 Gb of info to delete but I did not check it.

**Comment by Silarn** [comment]
Okay - then the issue is just downloads being recycled for some unknown reason. The downloads that are in your directory are showing up as expected. Unfortunately, I don't know how to replicate this issue, which makes it hard to diagnose much less fix. Which is kind of why we need logs of it actually happening or steps to reproduce the problem.
